Specs/DataSources/KmlDataSourceSpec.js

```diff
@@ -3714,4 +3716,29 @@ defineSuite([
             expect(console.log).toHaveBeenCalledWith('KML - refreshMode of onExpire requires the NetworkLinkControl to have an expires element');
         });
     });
+
+    it('NetworkLink: Heading and pitch can be undefined if the camera is in morphing mode', function() {
+        var kml = '<?xml version="1.0" encoding="UTF-8"?>\
+          <NetworkLink id="link">\
+            <Link>\
+              <href>./Data/KML/simple.kml</href>\
+              <refreshMode>onExpire</refreshMode>\
+            </Link>\
+          </NetworkLink>';
+
+        var oldMode = options.camera._mode;
+        var oldHeading = options.camera.heading;
+        var oldPitch = options.camera.pitch;
+
+        options.camera._mode = SceneMode.MORPHING;
+        options.camera.heading = undefined;
```

Comment:
On second thought, since this test can run asynchronously, I think we need to make a copy of options for this test instead.  Otherwise there may be a race condition that causes other tests to intermittently fail.